firebase-firestore/src/testUtil/java/com/google/firebase/firestore/testutil/TestUtil.java

```diff
@@ -431,12 +433,12 @@ public static PatchMutation patchMutation(
     boolean merge = updateMask != null;
 
     // We sort the fieldMaskPaths to make the order deterministic in tests.
-    Collections.sort(objectMask);
+    SortedSet<FieldPath> fieldMaskPaths = new TreeSet<>(merge ? updateMask : objectMask);
```

Comment:
Hrm.  I wonder if this is really necessary.  Were we just sorting them before so equals() would work out?  If so, then we're already good.  What happens if you use a HashSet here?

Reply:
Oh. I thought it was a HashSet.  In any case, I'm fine with using a TreeSet, but the comment about sorting seems confusing / wrong now.  Perhaps just remove it?

Reply:
Ah! Comment makes perfect sense now, thanks.